Modules/Files/test/Files.API.Tests.Performance.EndToEnd/src/utils.ts

```diff
@@ -50,16 +50,24 @@ export function getConfiguration(): Configuration {
   assertEnvVarExists("SIZE");
 
   return {
-    Host: __ENV.HOST,
+    Host: simplifyHost(__ENV.HOST),
     Client_Secret: __ENV.CLIENT_SECRET,
     Username: __ENV.USER,
     Password: __ENV.PASSWORD,
     Size: __ENV.SIZE,
   };
 }
 
-function assertEnvVarExists(parameter: string) {
+function assertEnvVarExists(parameter: string): void {
   if (!__ENV[parameter]) {
     exec.test.abort(`Parameter '${parameter}' cannot be null or empty`);
   }
 }
+
+function simplifyHost(host: string): string {
+  if (host.charAt(host.length - 1) == "/") {
```

Comment:
Always use `===` instead of `==` in Java Script (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)